src/multi_stage/dual_dynamic_programming.jl

```diff
@@ -200,6 +200,14 @@ function run_ddp(outpath::AbstractString, models_d::Dict, setup::Dict, inputs_d:
             t = 1 #  update forward pass solution for the first stage
             models_d[t], solve_time_d[t] = solve_model(models_d[t], setup)
             inputs_d[t]["solve_time"] = solve_time_d[t]
+
+            if myopic && mysetup["MultiStageSettingsDict"]["WriteIntermittentOutputs"]
+                for p in 1:mysetup["MultiStageSettingsDict"]["NumStages"]
+                    outpath_cur = joinpath(outpath, "results_p$p")
+                    write_outputs(model_dict[p], outpath_cur, mysetup, inputs_dict[p])
+                end
+            end
```

Comment:
This part should go after line 172, as for the myopic case `ic = 1`.

Reply:
Thanks for pointing that out, I just moved these lines to line 173.